packages/main/src/Table.ts

```diff
@@ -475,8 +460,6 @@ class Table extends UI5Element {
 	 * <br><br>
 	 * <b>Note:</b> Use <code>ui5-table-column</code> for the intended design.
 	 *
-	 * @type {sap.ui.webc.main.ITableColumn[]}
-	 * @name sap.ui.webc.main.Table.prototype.columns
 	 * @slot
 	 * @public
```

Comment:
TableColumn has template so we have to provide interface. Same applies to for table cell